tasks/omni.ts

```diff
@@ -0,0 +1,80 @@
+import { task, subtask } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import * as fs from "fs";
+import * as path from "path";
+import * as handlebars from "handlebars";
+import { upperFirst } from "lodash";
+
+async function processTemplates(
```

Comment:
I just notice, in general, we are using arrow functions, any good reason to use this instead?

Reply:
indeed, the main is arrow, I prefer to be consistent if both are ok

Reply:
Sure, I'll switch to arrows 👍

Reply:
https://github.com/zeta-chain/toolkit/pull/2/commits/36e999800fcdadb2e60c9d059f7d0276681d4c7d